app/models/docker_builder_service.rb

```diff
@@ -71,9 +80,12 @@ def build_image(tmp_dir)
     output.puts("### Running Docker build")
 
```

Comment:
it will fail nicely ;)

On Wed, Aug 17, 2016 at 11:59 AM, Jon Moter <notifications@github.com>
wrote:

> In app/models/docker_builder_service.rb
> <https://github.com/zendesk/samson/pull/1203#discussion_r75185103>:
>
> > @@ -71,9 +80,12 @@ def build_image(tmp_dir)
> >      output.puts("### Running Docker build")
> >
>
> Before we attempt a build, should we validate that the Dockerfile exists?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1203/files/efc715ffd4d282b94fec671f6aaa56511202dc73#r75185103>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZzQrjhaCWq3YEI2uSKW6u6YDW7XOks5qg1n1gaJpZM4JkwAM>
> .
>